virtcontainers/mount.go

```diff
@@ -330,23 +332,23 @@ func bindUnmountContainerRootfs(ctx context.Context, sharedDir, sandboxID, cID s
 	defer span.Finish()
 
 	rootfsDest := filepath.Join(sharedDir, sandboxID, cID, rootfsDir)
-	syscall.Unmount(rootfsDest, syscall.MNT_DETACH)
-
-	return nil
+	return syscall.Unmount(rootfsDest, syscall.MNT_DETACH)
```

Comment:
@gabibeyer I think we should also distinguish between general errors while unmounting and an error where the mount itself does not exist.
If the mount does not exist, we can simply log this and ignore this error.

Reply:
good idea.

Reply:
Okay, FWIW the kata_agent, on stop container attempts to unmount the rootfs and performs error handling, so if we do logging at this lower level and return nil for missing file/dir, it will affect that. do we care? Only asking since changing logic here may have side effects and I want to be sure that is okay!